PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetPerformance.cxx

```diff
@@ -2395,12 +2477,26 @@ void AliAnalysisTaskEmcalJetPerformance::ComputeBackground()
     }
     
     // Compute delta pT, as a function of centrality
-    
-    // EMCal
+    // EMCal acceptance only charged component
```

Comment:
A general comment: I don't have time now to understand exactly what your new feature is doing and why. But can  you please make it a configurable option, defaulting to false? So that the code runs exactly as before otherwise? It is best if your studies are a bit separated from the "standard" background computations that are already implemented, both for code readability/comprehension and so that a bunch of histograms aren't unnecessarily duplicated (which I think would be the case in what you've implemented, if I run both a charged and full jet container).

Reply:
I have added now the flag fFillChargedFluctuations that enables the charged fluctuations to be filled. The full fluctuations stay unaffected.

Reply:
Thanks